intel_extension_for_transformers/llm/runtime/graph/scripts/ci/cpp_graph_inference.sh

```diff
@@ -0,0 +1,399 @@
+        model_name="tiiuae/falcon-7b"
+        input_model="/tf_dataset2/models/nlp_toolkit/falcon-7b"
+    elif [[ "${model}" == "starcoder-3b" ]]; then
+        quant_script="./build/bin/quant_starcoder"
+        infer_cmd="./build/bin/run_starcoder"
+        model_name="bigcode/starcoder"
+        input_model="/tf_dataset2/models/pytorch/starcode_3b"
+    elif [[ "${model}" == "bloom-7b" ]]; then
+        quant_script="./build/bin/quant_bloom"
+        infer_cmd="./build/bin/run_bloom"
+        model_name="bigscience/bloom-7b1"
+        input_model="/tf_dataset2/models/pytorch/bloom-7b1"
+    elif [[ "${model}" == "opt-1.3b" ]]; then
+        quant_script="./build/bin/quant_opt"
+        infer_cmd="./build/bin/run_opt"
+        model_name="facebook/opt-1.3b"
+        input_model="/tf_dataset2/models/pytorch/opt-1.3b"
+    elif [[ "${model}" == "dolly-v2-3b" ]]; then
+        quant_script="./build/bin/quant_dolly"
+        infer_cmd="./build/bin/run_dolly"
+        model_name="databricks/dolly-v2-3b"
+        input_model="/tf_dataset2/models/pytorch/dolly_v2_3b"
+    elif [[ "${model}" == "chatglm2" ]]; then
+        quant_script="./build/bin/quant_chatglm2"
+        infer_cmd="./build/bin/run_chatglm2"
+        model_name="THUDM/chatglm2-6b"
+        input_model="/tf_dataset2/models/pytorch/chatglm2-6b"
+    elif [[ "${model}" == "chatglm-6b" ]]; then
+        quant_script="./build/bin/quant_chatglm"
+        infer_cmd="python ./scripts/inference.py"
+        model_name="THUDM/chatglm-6b"
+        input_model="/tf_dataset2/models/pytorch/chatglm-6b"
+        extension=" --model_name chatglm --tokenizer $input_model"
+        requirements_file="scripts/requirements/chatglm-6b.txt"
+    elif [[ "${model}" == "baichuan2-13b" ]]; then
+        quant_script="./build/bin/quant_baichuan"
+        infer_cmd="python ./scripts/inference.py"
+        model_name="baichuan-inc/Baichuan2-13B-Chat"
+        input_model="/tf_dataset2/models/pytorch/Baichuan2-13B-Chat"
+        extension=" --model_name baichuan --tokenizer $input_model"
+    elif [[ "${model}" == "baichuan-13b" ]]; then
+        quant_script="./build/bin/quant_baichuan"
+        infer_cmd="python ./scripts/inference.py"
+        model_name="baichuan-inc/Baichuan-13B-Chat"
+        input_model="/tf_dataset2/models/pytorch/Baichuan-13B-Chat"
+        extension=" --model_name baichuan --tokenizer $input_model"
+    elif [[ "${model}" == "mistral-7b" ]]; then
+        quant_script="./build/bin/quant_mistral"
+        infer_cmd="./build/bin/run_mistral"
+        model_name="mistralai/Mistral-7B-v0.1"
+        input_model="/tf_dataset2/models/pytorch/Mistral-7B-v0.1"
+        requirements_file="scripts/requirements/mistral.txt"
+    elif [[ "${model}" == "qwen-7b" ]]; then
+        quant_script="./build/bin/quant_qwen"
+        infer_cmd="./build/bin/run_qwen"
+        model_name="Qwen/Qwen-7B-Chat"
+        input_model="/tf_dataset2/models/nlp_toolkit/Qwen-7B-Chat"
```

Comment:
absolute path should never be part of project but can be part of CI

Reply:
Agree. Tried to fix in 8d6de3fc957a3f09dbd5fbbf046fc66d527e94b3.